shared/src/packages/orders/libs/types/order-create-request-dto.type.ts

```diff
@@ -2,7 +2,14 @@ import { type OrderEntity } from './order-entity.type.js';
 
 type OrderCreateRequestDto = Omit<
   OrderEntity,
-  'id' | 'userId' | 'businessId' | 'price' | 'status' | 'driver'
->;
+  | 'id'
+  | 'userId'
+  | 'businessId'
+  | 'price'
+  | 'status'
+  | 'driver'
+  | 'shift'
+  | 'truck'
```

Comment:
If we keep Omit:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-towhub/assets/92586731/5ddcd345-4ddb-4bca-b263-25bed52857c8)

So, it's better to make it Pick<>, because:
1. There're only 6 properties to pick instead of 8;
2. It will be clearer on what properties are here, because now, to understand it, we have to omit many properties, and still there'll be a lot of properties left.